test_runner/src/main/kotlin/ftl/util/Utils.kt

```diff
@@ -68,6 +69,13 @@ private fun getResource(name: String): InputStream {
         ?: throw FlankGeneralError("Unable to find resource: $name")
 }
 
+fun printVersionInfo() {
+    logLn("version: " + readVersion())
```

Comment:
```suggestion
    logLn("version:${readVersion}")
```
?
The same for all 2 lines below?

Reply:
Thanks! Changed!